cnv/cnv_users_guide/cnv-dashboard.adoc

```diff
@@ -0,0 +1,13 @@
+[id="cnv-dashboard"]
```

Comment:
This is a bit of a nit but it's good practice for the id to correlate to the title. It makes it easier to navigate content in our our repo (especially when cross-referencing from published doc), and is becoming increasingly important as our content grows. If you do decide to update the id, don't forget to update the filename to match.

Reply:
Hi Andrew,

I synced up file name and ID on the other file. cnv-dashboard looks fine to me (the file is cnv-dashboard.adoc). See below. Am I missing something? Thanks,

Bob

[id="cnv-dashboard"]
= Using the overview dashboard to get cluster information
include::modules/cnv-document-attributes.adoc[]
:context: cnv-dashboard
toc::[]

Access the overview dashboard, which captures high-level information about the cluster, by clicking *Home > Dashboards > Overview* from the {product-title} web console.

The overview dashboard provides various cluster information, captured in individual dashboard _cards_.

include::modules/cnv-web-console-overview-dashboard.adoc[leveloffset=+1]

Reply:
I meant the title of the assembly. In this case if the assembly is 'Using the overview dashboard to get cluster information' then I would give the id and filename something like 'cnv-using-overview-dashboard'. It's not a dealbreaker but it does make it much easier to find an assembly/module in the repo when you're looking at something in the built doc (subsequent releases/bugs).

Reply:
Given the time left in the release, I think I'll keep the assembly title as cnv-dashboard, but I will make a task to change the title after we release.

Reply:
Hey @bgaydosrh - doing this after the release means that the link to this assembly will change, if you change the ids and filenames. These will then require a redirect. It might be best to do this before.

Reply:
Gotcha, I will do this tomorrow.